ios/FluentUI/People Picker/ContactView.swift

```diff
@@ -0,0 +1,189 @@
+            avatarView.widthAnchor.constraint(equalToConstant: Constants.avatarViewWidth),
+            avatarView.centerXAnchor.constraint(equalTo: centerXAnchor),
+            avatarView.topAnchor.constraint(equalTo: topAnchor),
+            avatarView.leadingAnchor.constraint(equalTo: leadingAnchor),
+            avatarView.trailingAnchor.constraint(equalTo: trailingAnchor)
+        ]
+    }
+
+    private func titleLabelLayoutConstraints() -> [NSLayoutConstraint] {
+        guard let titleLabel = titleLabel else {
+            return []
+        }
+
+        titleLabel.translatesAutoresizingMaskIntoConstraints = false
+        return [
+            titleLabel.topAnchor.constraint(equalTo: avatarView.bottomAnchor, constant: Constants.spacingBetweenAvatarAndFirstLabel),
+            titleLabel.leadingAnchor.constraint(equalTo: leadingAnchor),
+            titleLabel.trailingAnchor.constraint(equalTo: trailingAnchor),
+            titleLabel.centerXAnchor.constraint(equalTo: centerXAnchor),
+            titleLabel.widthAnchor.constraint(equalTo: widthAnchor),
+            titleLabel.heightAnchor.constraint(greaterThanOrEqualToConstant: Constants.titleLabelMinimumHeight)
+        ]
+    }
+
+    private func subtitleLabelLayoutConstraints() -> [NSLayoutConstraint] {
+        guard let subtitleLabel = subtitleLabel else {
+            return []
+        }
+
+        subtitleLabel.translatesAutoresizingMaskIntoConstraints = false
+
+        var constraints = [
+            subtitleLabel.leadingAnchor.constraint(equalTo: leadingAnchor),
+            subtitleLabel.trailingAnchor.constraint(equalTo: trailingAnchor),
+            subtitleLabel.centerXAnchor.constraint(equalTo: centerXAnchor),
+            subtitleLabel.widthAnchor.constraint(equalTo: widthAnchor),
+            subtitleLabel.bottomAnchor.constraint(equalTo: bottomAnchor)
+        ]
+
+        if let titleLabel = titleLabel {
+            constraints.append(subtitleLabel.topAnchor.constraint(equalTo: titleLabel.bottomAnchor))
+        }
+
+        return constraints
+    }
+
+    private func identifierLayoutConstraints() -> [NSLayoutConstraint] {
+        guard let titleLabel = titleLabel else {
+            return []
+        }
+
+        titleLabel.translatesAutoresizingMaskIntoConstraints = false
+        return [
+            titleLabel.widthAnchor.constraint(equalTo: widthAnchor),
+            titleLabel.centerXAnchor.constraint(equalTo: centerXAnchor),
+            titleLabel.topAnchor.constraint(equalTo: avatarView.bottomAnchor, constant: Constants.spacingBetweenAvatarAndFirstLabel),
+            titleLabel.leadingAnchor.constraint(equalTo: leadingAnchor),
+            titleLabel.trailingAnchor.constraint(equalTo: trailingAnchor),
+            titleLabel.bottomAnchor.constraint(equalTo: bottomAnchor)
+        ]
+    }
+
+    private func setupTitleLabel(using firstName: String) {
+        titleLabel = UILabel(frame: .zero)
+
+        if let titleLabel = titleLabel {
```

Comment:
> if let titleLabel = titleLabel { [](start = 7, length = 33)

it's kind of weird to do this when we just initialized it in the line above. Instead you could do:
let label = UILabel(...)
// configure label
titleLabel = label

Reply:
I see what you mean. I initialized it and then immediately checked whether or not it was nil which is redundant.